src/main/scala/nl/knaw/dans/easy/multideposit/actions/AddDatasetMetadataToDeposit.scala

```diff
@@ -167,6 +167,9 @@ object AddDatasetMetadataToDeposit {
         case (Some(q), None,   Some(t)) => elem(s"dcterms:$q")(t)
         case (None,    Some(l),_      ) => elem(s"dc:relation")(l)
         case (None,    None,   Some(t)) => elem(s"dc:relation")(t)
+        case _                          =>
+          // TODO this case needs to be checked to not occur in the preconditions (see also line 32)
```

Comment:
It requires a bit of browsing through github (history an view button) or figuring out a commit SHA otherwise and assemble the link manually. You don't need the full SHA and can specify a range of lines with #L31-#l33 not sure a commit SHA might get lost with a squash merge and dropped branches, but github might preserve commits of a pull request even when its branch is dropped.

https://github.com/rvanheest/easy-split-multi-deposit/blob/6f17388233360f47f1616f42e458f5fee461987b/src/main/scala/nl/knaw/dans/easy/multideposit/actions/AddDatasetMetadataToDeposit.scala#L32